docs/en/trust_marks_spid.rst

```diff
@@ -0,0 +1,95 @@
+.. include:: ./common_definitions.rst
+
+
+Trust Marks for SPID
+++++++++++++++++++++
+
+
+Examples of Trust Marks for SPID
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The TMs issued for the Leaves MUST be published by the Leaves themselves in their own **Entity Configuration**,
```

Comment:
Leaves -> Leafs

please do a global check for this typo

Reply:
No. The plural of Leaf is "Leaves"